test/mix_task_test.exs

```diff
@@ -0,0 +1,10 @@
+defmodule Mix.Tasks.TicTacToe.Test do
+  use ExUnit.Case
+  import ExUnit.CaptureIO
+
+  test "Plays the game once" do
```

Comment:
consistency... here you're starting the test description with upper case but all the other test descriptions are lower case.